src/maheshKJun20/StringExamples/SumofDigitInStringUsingAscii.java

```diff
@@ -0,0 +1,22 @@
+package maheshKJun20.StringExamples;
+
+public class SumofDigitInStringUsingAscii {
+	
+	//Method to calculate sum of all digits in a String using Ascii value
+	void displaySumOfDigits(String str) {
+		int sum = 0;
+		for (int index=0;index<str.length();index++) {
+			int asciiValue = str.charAt(index);
+			if (asciiValue >= 48  && asciiValue <= 57)
+				sum += Character.getNumericValue(str.charAt(index));
```

Comment:
Good @MaheshKJun20 

You can use below logic as well
sum += (str.charAt(index) - '0');
Or sum = sum + Character.getNumericValue(str.charAt(index);
or sum += Integer.parseInt(String.valueOf(input.charAt(index)));